python/tvm/relay/frontend/tflite.py

```diff
@@ -982,13 +982,15 @@ def convert_fully_connected(self, op):
 
         weight_value = self.get_tensor_value(weight_tensor)
         weight_expr = self.exp_tab.new_const(weight_value, dtype=weight_tensor_type_str)
+        weight_shape = _infer_shape(weight_expr)
 
         if input_tensor.qnn_params:
             out = _qnn.op.dense(in_expr, weight_expr,
                                 input_zero_point=input_tensor.qnn_params['zero_point'],
                                 kernel_zero_point=weight_tensor.qnn_params['zero_point'],
                                 input_scale=input_tensor.qnn_params['scale'],
                                 kernel_scale=weight_tensor.qnn_params['scale'],
+                                units=weight_shape[1],
```

Comment:
can you confirm if this change is correct? This code path is not run during tflite forntend tests. @anijain2305

Reply:
Should it weight_shape[0]. Dense input shapes are m * k and n * k.
And units means n here, right?

Reply:
I think units is supposed to mean output dimension (the dimension that weight scale is applied)

From the test cases in https://github.com/apache/incubator-tvm/blob/70c63829474649f34928d42948bf3e5c7dbfbd75/tests/python/frontend/tflite/test_forward.py#L1381-L1384, I guessed that `weight_shape[1]` is the output dimension.

Reply:
fixed

Reply:
weight_shape[0] should be the output dimension. The script you see is tf's layout, which is not the same as tflite.